code/colonial marines/unk379.dm

```diff
@@ -166,11 +166,13 @@
 			icon_state = "desert[rand(1,5)]"
 		else
 			icon_state = "desert_dug"
-	if(prob(1))
-		if(prob(50))
-			new /obj/effect/overlay/palmtree_l(src)
-		else
-			new /obj/effect/overlay/palmtree_r(src)
+
+	if (ticker.state != GAME_STATE_PLAYING)
```

Comment:
if it's an attempted fix for palmtrees spawning on any kind of grass new turf, no need,i already fixed it (IN THE PROPER WAY)